### ISSUE TITLE
null ref in ConfigureAwaitAnalyzer

### ISSUE THREAD

**Issue by SimonCropp** [Particular/Particular.CodeRules#8]
```
Severity	Code	Description	Project	File	Line	Suppression State	Detail Description
Error	AD0001	Analyzer 'Particular.CodeRules.ConfigureAwait.ConfigureAwaitAnalyzer' threw an exception of type 'System.NullReferenceException' with message 'Object reference not set to an instance of an object.'.	NServiceBus.Core(netstandard2.0)		1	Active	Analyzer 'Particular.CodeRules.ConfigureAwait.ConfigureAwaitAnalyzer' threw the following exception:
'Exception occurred with following context:
Compilation: NServiceBus.Core
SyntaxTree: C:\Code\Particular\NServiceBus\src\NServiceBus.Core\Persistence\Learning\LearningSynchronizedStorageSession.cs
SyntaxNode: await sagaStorageFile.Read [AwaitExpressionSyntax]@[1290..1316) (45,31)-(45,57)

System.NullReferenceException: Object reference not set to an instance of an object.
   at Particular.CodeRules.ConfigureAwait.ConfigureAwaitAnalyzer.AnalyzeForConfigureFalse(SyntaxNodeAnalysisContext context)
   at Microsoft.CodeAnalysis.Diagnostics.AnalyzerExecutor.<>c__43`1.<ExecuteSyntaxNodeAction>b__43_0(ValueTuple`2 data)
   at Microsoft.CodeAnalysis.Diagnostics.AnalyzerExecutor.ExecuteAndCatchIfThrows_NoLock[TArg](DiagnosticAnalyzer analyzer, Action`1 analyze, TArg argument, Nullable`1 info)
-----
'.
```

**Comment by timbussmann** [comment]
hey @SimonCropp , could this be due to the same issue as described in this issue: https://github.com/Particular/Particular.CodeRules/issues/4 ?

**Comment by timbussmann** [comment]
👍 

fyi: there's a PR to fix the NRE here: https://github.com/Particular/Particular.CodeRules/pull/9

**Comment by timbussmann** [comment]
#9 has been fixed as part of 0.2.1. I'm going to close this issue as I think this is caused due to the same bug. Please feel free to reopen in case you continue to see an NRE. 

Thanks for raising this issue, the exception information was definitely helpful 👍
